router/middlewares/middlewareLogger.go

```diff
@@ -5,11 +5,16 @@ import (
 	"net/http"
 
 	"github.com/ditrit/badaas/configuration"
+	"github.com/gorilla/mux"
 	"github.com/noirbizarre/gonja"
 	"github.com/noirbizarre/gonja/exec"
 	"go.uber.org/zap"
 )
 
+func AddLoggerMiddleware(router *mux.Router, middlewareLogger MiddlewareLogger) {
```

Comment:
Remove this changes from commit "fix typo in Authentication" because it's not a typo.